### PR TITLE
Adding VerifyTabletAssignments utility to accumulo admin command

### PR DESCRIPTION
**Pull request by cshannon**
Issue #2807

### CODE REVIEW

**Comment by milleruntime** [comment]
We have a `checkTablets` command under Admin that may be similar to this utility. I am going to look at it more closely but I think this could go under Admin as well.

**Comment by cshannon** [comment]
> We have a `checkTablets` command under Admin that may be similar to this utility. I am going to look at it more closely but I think this could go under Admin as well.

Ok sounds good, I can move this to Admin when I get some time later this week.

**Comment by cshannon** [comment]
@milleruntime - This has been updated to move to the Admin command

File: server/base/src/main/java/org/apache/accumulo/server/util/Admin.java [hunk 1]
```diff
@@ -162,6 +162,13 @@ static class RandomizeVolumesCommand {
     String tableName = null;
   }
 
+  @Parameters(commandDescription = "Verify tablet assignments")
```
Comment (milleruntime) [inline server/base/src/main/java/org/apache/accumulo/server/util/Admin.java:165]: Description could be better... What about "Verify all Tablets are assigned to tablet servers"

Comment (cshannon) [reply server/base/src/main/java/org/apache/accumulo/server/util/Admin.java]: @milleruntime - This has been updated

**Review by Manno15 — APPROVED** [review]
Changes look good from a performance standpoint. Everything worked as expected.

**Review by milleruntime — APPROVED** [review]

**Pull request merged**